internal/docker/option.go

```diff
@@ -0,0 +1,29 @@
+package docker
+
+//GetOpts - iterate the inbound Options and return a struct.
+func GetOpts(opt ...Option) Options {
+	opts := getDefaultOptions()
+	for _, o := range opt {
+		o(&opts)
+	}
+	return opts
+}
+
+// Option - how Options are passed as arguments.
+type Option func(*Options)
+
+// Options - how Options are represented.
+type Options struct {
+	withContainerImage string
+}
+
+func getDefaultOptions() Options {
+	return Options{}
+}
+
```

Comment:
The double newline here is a clue to me that you don't have your editor formatting code for you on save 😸